gremlin-server/pom.xml

```diff
@@ -59,6 +59,16 @@ limitations under the License.
             <artifactId>logback-classic</artifactId>
             <optional>true</optional>
         </dependency>
+        <dependency>
```

Comment:
Note: This package is Apache 2 licensed with no provided NOTICE file. I see no concerns from a licensing standpoint.